.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 10.22.0
```

Comment:
Did you mean to commit this file? We officially recommend nvm for Node.js, not asdf.

Reply:
nope 😊 
Sorry for that